lib/libc/machine/arm/armv7-m/arch_elf.c

```diff
@@ -193,7 +193,7 @@ int up_relocate(FAR const Elf32_Rel *rel, FAR const Elf32_Sym *sym, uintptr_t ad
 	}
 	break;
 
-#ifdef CONFIG_ARMV7M_TARGET2_PREL
+#if defined(CONFIG_ARMV7M_TARGET2_PREL) || defined(CONFIG_ARMV8M_TARGET2_PREL)
```

Comment:
Is  arch_elf.c module is common for armv7-m & armv8-m?

Reply:
Yes, this file is same for both.

Reply:
What do you mean the "same"? This file is in the `armv7-m`. Do you mean that armv8-m chip uses armv7-m's file?

Reply:
https://github.com/Samsung/TizenRT/blob/72bf01c06731f84bfcab4235fb67c37898966489/lib/libc/machine/arm/Make.defs#L62-L65

Yes, both use the same file.

Reply:
It looks strange. No need to move this into the common? Or make a common for m core if r core is different?

Reply:
Need to check this